cmd/dashboard/docker/Dockerfile

```diff
@@ -48,18 +51,9 @@ RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build \
 # Output stage: Creates version file, copies binary and assets to an alpine image
 #
 
-FROM nginx:stable-alpine
+FROM $NGINX_IMAGE
 
-ARG DOCKER_CLI_VERSION="19.03.12"
-
-ENV DOWNLOAD_URL="https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_CLI_VERSION.tgz"
-
-RUN apk --update --no-cache add ca-certificates git curl parallel \
-    && mkdir -p /tmp/download \
-    && curl -L $DOWNLOAD_URL | tar -xz -C /tmp/download \
-    && mv /tmp/download/docker/docker /usr/local/bin/ \
-    && rm -rf /tmp/download \
-    && apk del curl
+RUN apk --update --no-cache add ca-certificates git parallel docker-cli
```

Comment:
this will unlock the previously locked dependency of docker client

Reply:
I know, just another unlocked dependency like parallel, git and so on, given free by alpine repo hub
and release us from trying to find the appropriate arch link
I made sure their package version aligned with our current one

Reply:
Update: decided to lock version again